builtin/logical/consul/path_roles.go

```diff
@@ -157,17 +176,22 @@ func (b *backend) pathRolesWrite(ctx context.Context, req *logical.Request, d *f
 	policy := d.Get("policy").(string)
 	policies := d.Get("policies").([]string)
 	roles := d.Get("consul_roles").([]string)
+	serviceIdentities := d.Get("service_identities").(string)
```

Comment:
Should we do any sort of validation in here, especially for `service_identities` since the input is a just string?

Reply:
I'm not quite sure how you would validate the entry for these. Since the name of a service or node can be anything and providing datacenters is optional, the presence of commas or semicolons is optional. Is there another form of validation I'm missing?